modules/robots.yml

```diff
@@ -12,5 +12,6 @@ maintainers:
   - name: Ricardo Gobbo de Souza
     github: ricardogobbosouza
 compatibility:
-  nuxt: ^2.0.0
-  requires: {}
+  nuxt: ^2.0.0 || ^3.0.0
```

Comment:
```suggestion
  nuxt: ^2.0.0 || ^3.0.0-rc.8
```

Reply:
Guess we need to upgrade all modules for this right?

Reply:
Yes indeed. Just wanted to mention we need to do this (also in review process check modules code)